src/Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -183,6 +184,8 @@ public void RegisterRules(IConsensus consensus)
                     new StratisBigFixPosFutureDriftRule(),
                     new CheckDifficultyPosRule(),
                     new StratisHeaderVersionRule(),
+                    new ProvenHeaderSizeRule(),
```

Comment:
Please order all the rules the same way they are ordered in the proven headers design document